content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -22,69 +22,102 @@
 
 In the following sections, we will cover the following topics:
 
->1. the typology of knowledge graphs (KG).
->2. the different technologies supporting them.
->3. the relation between FAIR and KGs.
->4. the methods to generate knowledge graphs.
```

Comment:
the original version touched this topic a bit. I think it can be a separate elaborated recipe